### DISCUSSION TITLE
Bug with lemmatisation: lemma of "cameras" is "cameras"?

### DISCUSSION THREAD

**Discussion by Saibo-creator** [explosion/spaCy#11873] [Help: Other Questions] [answered]
## How to reproduce the behaviour

```python
import spacy
nlp = spacy.load('en_core_web_sm')
doc = nlp("cameras, fungi")
for token in doc:
    print(token._lemma)
# return  cameras, fungi, expected: camera, fungus
```

## Info about spaCy

* **spaCy version:** 2.3.8
* **Platform:** macOS-10.16-x86_64-i386-64bit
* **Python version:** 3.8.13
* **Models:** en

**Comment by adrianeboyd** [comment, marked as answer]
The underlying cause is the same reason as described here: https://github.com/explosion/spaCy/discussions/5343#issuecomment-618348124

If your texts are longer with more context ("The photographer always carries two cameras with her when she photographs fungi in the woods.") then the POS is more likely to be correct.
